oak-api/src/main/java/org/apache/jackrabbit/oak/api/jmx/package-info.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-@Version("4.9.0")
+@Version("4.10.0")
```

Comment:
Do we need to bump this, if the change is in a providerType interface ?

Reply:
Build is failing without minor bump. Adding providerType doesn't mean that version bump is not needed.